File: CodeWars/JavaScript/7Kyu/Mumbling.js
// URL: https://www.codewars.com/kata/5667e8f4e3f572a8f2000039/train/javascript

function accum(s) {
  let mumbleString = "";
  
  for (i=0; i<s.length; i++){
    mumbleString += s[i].toUpperCase() + s[i].toLowerCase().repeat(i) + "-";
  }
  
  return mumbleString.slice(0,-1);
}

File: CodeWars/Java/7Kyu/Mumbling.java
// URL: https://www.codewars.com/kata/5667e8f4e3f572a8f2000039/train/java

public class Mumbling {

  public static String accum(String s) {
    String mumbleString = "";

    for (int i = 0; i < s.length(); i++) {
      mumbleString += Character.toString(s.charAt(i)).toUpperCase()
          + Character.toString(s.charAt(i)).toLowerCase().repeat(i) + "-";
    }

    return mumbleString.substring(0, mumbleString.length() - 1);
  }
}